app/frontend/src/components/Avatar/Avatar.tsx

```diff
@@ -1,40 +1,47 @@
-import {
-  Avatar as MuiAvatar,
-  AvatarProps as MuiAvatarProps,
-} from "@material-ui/core";
+import { Avatar as MuiAvatar, Box, BoxProps } from "@material-ui/core";
 import { makeStyles } from "@material-ui/core/styles";
 import classNames from "classnames";
 import React from "react";
 import { User } from "../../pb/api_pb";
 
 const useStyles = makeStyles({
-  root: { height: "3rem", width: "3rem" },
+  root: {
```

Comment:
I had to make changes as the initials in case of no profile picture weren't centered when 'grow' was used.